week 6.cpp

```diff
@@ -0,0 +1,204 @@
+#include <iostream>
+#include <stdlib.h>
+using namespace std;
+int id=0;
+class Item
+{
+	int item_id;
+	//char item_name;
+	float price;
+	
+	
+	public:
+
+	int item_count;
+	Item()
+	{
+		item_id=id++;
+		price=10;
+		item_count=2;
```

Comment:
These values should be parameters of the constructors so that we could have items with different price/quantities.